lib/std/event/loop.zig

```diff
@@ -503,7 +503,7 @@ pub const Loop = struct {
         }
     }
 
-    pub async fn bsdWaitKev(self: *Loop, ident: usize, filter: i16, fflags: u32) void {
+    pub async fn bsdWaitKev(self: *Loop, ident: usize, filter: i16, flags: u16) void {
```

Comment:
No async needed here (this is leftover from before the async/await rewrite)

```suggestion
    pub fn bsdWaitKev(self: *Loop, ident: usize, filter: i16, flags: u16) void {
```